### PR TITLE
Change comand handling

### PR DESCRIPTION
**Pull request by Fahl-Design**

### CODE REVIEW

File: src/IKEA/Tradfri/Helper/Online.php [hunk 1]
```diff
@@ -14,33 +14,47 @@ class Online
...
-     * @param string $ip
+     * @param string $ipAddress
      *
      * @return bool
      */
-    public static function isOnline(string $ip): bool
+    public static function isOnline(string $ipAddress): bool
     {
+        $online = true;
+        $regex = '/(\ \d+% packet loss)/';
```
Comment (synga-nl) [inline src/IKEA/Tradfri/Helper/Online.php]: This regex doesn't work on a mac. I would suggest: `(\ [\d\.]+% packet loss)`.

Example output of ping command:

PING xxxxxxxxx(192.168.1.15): 56 data bytes
64 bytes from xxxxxxxxx: icmp_seq=0 ttl=128 time=0.197 ms

--- xxxxxxxxx ping statistics ---
1 packets transmitted, 1 packets received, 0.0% packet loss
round-trip min/avg/max/stddev = 0.197/0.197/0.197/0.000 ms

File: src/IKEA/Tradfri/Helper/Online.php [hunk 2]
```diff
@@ -14,33 +14,47 @@ class Online
...
+     *
+     * @param array $matches
+     *
+     * @throws \IKEA\Tradfri\Exception\RuntimeException
+     *
+     * @return bool
+     */
+    protected static function _validateMatches(array $matches): bool
+    {
+        if (\strpos($matches[0][1], ' 0% packet loss') === false) {
```
Comment (synga-nl) [inline src/IKEA/Tradfri/Helper/Online.php]: This doesn't work on a Mac too, see previous comment

File: src/IKEA/Tradfri/Helper/Runner.php [hunk 3]
```diff
@@ -12,19 +12,25 @@
 class Runner
```
Comment (synga-nl) [inline src/IKEA/Tradfri/Helper/Runner.php:12]: My gateway can only handle a certain amount of concurrent requests, I would suggest to make a queue and use promises to return the data. For now I would suggest you can define a usleep time as a constant. So every call will wait x amount of time (configurable with the constant) before the next one is executed.

File: src/IKEA/Tradfri/Helper/Online.php (hunk 1 above)
Comment (Fahl-Design) [reply src/IKEA/Tradfri/Helper/Online.php]: Hi, I changed it to your suggestion thx!

File: src/IKEA/Tradfri/Helper/Runner.php (hunk 3 above)
Comment (Fahl-Design) [reply src/IKEA/Tradfri/Helper/Runner.php:12]: hi, I added your suggestion and will add a command queue soon. I have plans to change the command handling anyway. WIth an Invoker and a Receiver like https://github.com/domnikl/DesignPatternsPHP/tree/master/Behavioral/Command

**Pull request merged**